test/opsem2/bswap/bswap-ntohs-inline-asm_sat.c

```diff
@@ -0,0 +1,20 @@
+//; RUN: %sea "%s" 2>&1 | OutputCheck %s
+// CHECK: ^Error: assertion failed
+// CHECK: ^sat$
+
+#include "seahorn/seahorn.h"
+#include <arpa/inet.h>
```

Comment:
is this available on all linux systems... or will we have to install in Continuous integration server which runs ubuntu?